### PR TITLE
#164198153 Share articles

### PR DESCRIPTION
**Pull request by davdwhyte87**
**What does this PR**
This PR enables a user to share an article

**Tasks to be completed?**
- add components with social media buttons
- get current article link
- compose social media share link in href

**How can this be manually tested?**
- clone the repo
- install node js and yarn
- run `yarn install`
- run `yarn start:dev`
- click on an article and share it

**Relevant PT stories?**
`164198153`

**Screenshots**
SharePR

### CODE REVIEW

File: src/components/SocialShare.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * This component contains all the share buttons
+ * @class SocialShare
+ * @extends { React.Component }
+ */
+export default function SocialShare() {
+  const articleLink = window.location.href;
+  return (
+    <div>
+      <a href={`https://mail.google.com/mail/u/0/?view=cm&fs=1&to&su=Authors+Haven&body=${articleLink}`} className="text-danger" rel="noopener noreferrer" target="_blank"><i className="fas fa-envelope" /></a>
```
Comment (aanchirinah) [inline src/components/SocialShare.jsx:12]: #### Nitpick
- Do this better please , it's against DRY.

**Review by aanchirinah — CHANGES_REQUESTED** [review]
Good job @davdwhyte87 , attend to my requested changes.

File: src/styles/shared.scss [hunk 2]
```diff
@@ -8,6 +8,9 @@ body {
 	background-color: colorpurple;
 	color: $white;
 }
+.pg {
```
Comment (fejiroofficial) [inline src/styles/shared.scss]: I strongly believe you can find a better name for this selector.

File: src/views/ArticleBody.jsx [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Tags from './ArticleTags';
 import clapImage from '../images/clap.svg';
+import SocailShare from '../components/SocialShare';
```
Comment (fejiroofficial) [inline src/views/ArticleBody.jsx]: #### Nitpick
Incorrect and inconsistency in spelling can introduce an unwanted bug to your code base. Kindly rename this to properly represent what it is doing.

**Review by fejiroofficial — COMMENTED** [review]
Good job @davdwhyte87 Kindly attend to my requests.

File: src/views/ArticleBody.jsx (hunk 3 above)
Comment (davdwhyte87) [reply src/views/ArticleBody.jsx]: Yes thanks for that. I have changed it. https://github.com/andela/apollo-ah-frontend/blob/feature%2F164198153%2Fshare-article/src%2Fviews%2FArticleBody.jsx/#L5

File: src/styles/shared.scss (hunk 2 above)
Comment (davdwhyte87) [reply src/styles/shared.scss]: Alright. https://github.com/andela/apollo-ah-frontend/blob/feature%2F164198153%2Fshare-article/src%2Fstyles%2Fshared.scss/#L11-L13

**Comment by aanchirinah** [comment]
@davdwhyte87 You have code climate violations.

**Pull request closed without merging**